iis-wf/iis-wf-export-actionmanager/src/main/resources/eu/dnetlib/iis/wf/export/actionmanager/sequencefile/oozie_app/workflow.xml

```diff
@@ -291,6 +291,7 @@
             </main-class>
             <arg>eu.dnetlib.iis.referenceextraction.project.schemas.DocumentToProject</arg>
             <arg>eu.dnetlib.iis.referenceextraction.dataset.schemas.DocumentToDataSet</arg>
+            <arg>eu.dnetlib.iis.referenceextraction.service.schemas.DocumentToService</arg>
```

Comment:
How it worked before without DocumentToDatasource here?

Reply:
Apparently this was required in one of the previous versions of CDH, when this `generate-schema` approach was introduced, but then it turned out to be redundant at some point.

Last week I have created a dedicated issue for this, #1415, as a follow-up task I will probably pick up in the next sprint.